.github/workflows/stale.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Run stale action
-      uses: actions/stale@v7.0.0
```

Comment:
This changes the name and version number. Given we use `actions/checkout` elsewhere at a version `3.5.2` this seems wrong.

Reply:
Good catch. Fixed.